app/views/users/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_tag "/profile", method: :patch, class: "edit-user-form" do %>
+  <%= tag.h1 "Edit User Profile" %>
+
+  <%= label_tag :name, "Full Name" %>
+  <%= text_field_tag :name, @user.name %>
+
+  <%= label_tag :address, "Street Address" %>
+  <%= text_field_tag :address, @user.address %>
+
+  <%= label_tag :city, "City" %>
+  <%= text_field_tag :city, @user.city %>
+
+  <%= label_tag :state, "State" %>
+  <%= text_field_tag :state, @user.state %>
+
+  <%= label_tag :zip, "ZIP Code" %>
+  <%= number_field_tag :zip, @user.zip %>
+
+  <%= label_tag :email, "Email Address" %>
+  <%= text_field_tag :email, @user.email %>
+
```

Comment:
Nice design decision to not include a password field to verify.
It makes the user experience better and the `on: :create` callback is sly.